### ISSUE TITLE
Please bump `commander` dependency to avoid `abbrev` warning on Ruby 3.3

### ISSUE THREAD

**Issue by owst** [AaronC81/parlour#139]
**Is your feature request related to a problem? Please describe.**

Parlour transitively depends on a version of highline that emits a warning on Ruby 3.3.

The [latest release](https://rubygems.org/gems/parlour/versions/8.1.0) of parlour depends on `commander ~> 4.5`, [which](https://rubygems.org/gems/commander/versions/4.6.0) depends on `highline ~> 2.0.0`, which emits a warning on Ruby 3.3 when required:
```
/Users/owenstephens/.rbenv/versions/3.3.0/lib/ruby/gems/3.3.0/gems/highline-2.0.3/lib/highline.rb:17: warning: abbrev was loaded from the standard library, but will no longer be part of the default gems since Ruby 3.4.0. Add abbrev to your Gemfile or gemspec. Also contact author of highline-2.0.3 to add abbrev into its gemspec.
```

**Describe the solution you'd like**

Please change the `commander` dependency to `~> 5.0` to release a transitive dependency on `highline` so that the fix in https://github.com/JEG2/highline/pull/268 can be included (which was released in [highline 3.0.1](https://rubygems.org/gems/highline/versions/3.0.1))

The commander changelog for 5.0.0 mentions that moving to 5.0 will drop support for Ruby < 3 and update the highline dependency: 

https://github.com/commander-rb/commander/blob/98dee54912730397d06eca9132dbfd2ec3bbd4b0/History.rdoc#L1-L3

**Describe alternatives you've considered**

I think the requested change is reasonable since Ruby 2.7 (the last that is < 3) has been EOL for [over a year](https://www.ruby-lang.org/en/downloads/branches/)

**Additional context**

To reproduce, use Ruby 3.3.0, and a simple Gemfile that includes:
```
gem "parlour", "= 8.1.0"
```
then a warning will be printed as follows when requiring highline:
```
$ bundle exec ruby -rhighline -e 'puts "hello world from Ruby #{RUBY_VERSION}"'
/Users/owenstephens/.rbenv/versions/3.3.0/lib/ruby/gems/3.3.0/gems/highline-2.0.3/lib/highline.rb:17: warning: abbrev was loaded from the standard library, but will no longer be part of the default gems since Ruby 3.4.0. Add abbrev to your Gemfile or gemspec. Also contact author of highline-2.0.3 to add abbrev into its gemspec.
hello world from Ruby 3.3.0
```

**Comment by AaronC81** [comment]
@owst, @hlascelles - Apologies for the long delay. This is now released as Parlour 9.0.0, and removes the warning on my end!

**Comment by owst** [comment]
Thanks @AaronC81!
